src/app/telegram/votigram/components/VoteItem/index.css

```diff
@@ -1,18 +1,13 @@
 .telegram-vote-item {
-  @apply overflow-hidden relative bg-[#221D51];
   border-radius: 16px;
   border: 1px solid var(--Border-1, #403493);
-  .fake-content {
-    @apply h-[60px];
-  }
-  .bg {
-    z-index: 1;
-    border-radius: 16px;
-    @apply  h-[60px] w-full bg-[#4600C5] absolute left-0 top-[0px];
-  }
+  z-index: 1;
+  border-radius: 16px;
+  @apply overflow-hidden relative h-[60px] w-full bg-[#4600C5];
+
```

Comment:
In this case, it may not work.
![image](https://github.com/user-attachments/assets/8e736349-3416-40ad-bf9d-d0dd379a26ad)

Reply:
We will look into it later